app.js

```diff
@@ -8,11 +8,17 @@ const app = express()
 
 const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short'
 
+
+
+app.get('/', (req, res) => {
+  res.send('Hello World!');
+});
+
```

Comment:
Jeśli ten kod nie jest potrzebny to go usuń bo on nic nie robi i jeżeli nie planujesz go jakoś wykorzystać to lepiej go od razu usunąć bo potem będzie tylko przeszkadzał :)